src/traces/parcoords/defaults.js

```diff
@@ -49,6 +49,11 @@ function dimensionDefaults(dimensionIn, dimensionOut) {
     }
 
     var values = coerce('values');
+    if(!Array.isArray(values) && Lib.isArrayOrTypedArray(values)) {
+        // should convert typed arrays e.g. integers to real numbers
+        values = dimensionOut.values = Array.prototype.slice.call(values);
```

Comment:
Nop. Let's not do this during defaults to remain consistent with other typed-array-to-array blocks.

Move this to `parcoords/calc.js`.

Reply:
> If we can keep typed arrays as typed array and handle them correct downstream, then we shouldn't convert them.

That makes perfect sense!